plugins/linguist/package.json

```diff
@@ -10,7 +10,8 @@
   "exports": {
     ".": "./src/index.ts",
     "./alpha": "./src/alpha.ts",
-    "./package.json": "./package.json"
+    "./package.json": "./package.json",
+    "./src/translation": "./src/translation.ts"
```

Comment:
This should not be needed, please remove this.

Reply:
What you want to do is add this: `export * from './translation';`to this file: https://github.com/backstage/backstage/blob/master/plugins/linguist/src/alpha.ts

I'm just following what was done in the `user-settings` plugin which was the first plugin to add internationalization support: https://github.com/backstage/backstage/tree/master/plugins/user-settings

Reply:
This should also help with the API reports as well, I think at least

Reply:
Thank you, that fixes all!